### ISSUE TITLE
After overfitting training the inference result is bad

### ISSUE THREAD

**Issue by zhuobaidong** [FlyMyAI/flymyai-lora-trainer#44]
Hi there, thank you for your wonderful job!

before the whole datatset training, i just using 10 data pairs to train lora model, and i load this model for inference, but i found that the inference is bad and have a huge difference with training images. I was wondering that maybe the learning rate is too low to update the lora parameter, but i did not change the config content, has anyone confronting this problem?
